formatter/github.go

```diff
@@ -0,0 +1,39 @@
+package formatter
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// Github is an implementation of the Formatter interface
+// which formats the errors for GitHub Actions annotations.
+// https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message
+type Github struct {
+	Metadata lint.FormatterMetadata
+}
```

Comment:
GitHub is a brand with a specific case

```suggestion
type GitHub struct {
	Metadata lint.FormatterMetadata
}
```

Reply:
Thanks. Fixed